src/stylesheets/app.scss

```diff
@@ -0,0 +1,298 @@
+
+.pillbar {
+  background-image: linear-gradient(0deg, $florence-red 75%, $florence-blue 25%);
+  background-image: -webkit-linear-gradient(0deg, $florence-red 75%, $florence-blue 25%);
+  border-radius: 70px;
+  box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  color: #d2e7e2;
+  display: flex;
+  font-family: 'geomanist';
+  height: 35px;
+  margin: 5% auto;
+  width: 40%;
+
+  @media (max-width: $screen-sm) {
+    display: none;
+  }
+
+  a {
+    color: #d2e7e2;
+    text-decoration: none;
+  }
+
+  ul {
+    display: flex;
+    flex-direction: row;
+    font-size: 11px;
+    justify-content: space-around;
+    list-style: none;
+    margin: auto;
+    text-transform: uppercase;
+    width: 100%;
+  }
+}
+
+.box {
+
+  background: #ffffff;
+  border-radius: 4px;
+  height: 100%;
+  margin: auto;
+  max-width: 70%;
+  padding: 2% 5%;
+
+  @media (max-width: $screen-sm) {
+    max-width: 100%;
+    padding-bottom: 10%;
+  }
+}
+
+
+/*---------- First box ---------*/
+
+#box-one {
+  position: relative;
+}
+
+#main-thought {
+  font-family: 'Merriweather';
+  font-size: 1.7em;
+  line-height: 1.5em;
+
+  @media (max-width: $screen-sm) {
+    font-size: 1.4em;
+  }
+}
+
+.quote {
+  margin-left: 38%;
+  text-align: center;
+  width: 60%;
+}
+
+#quote-text
+  color: $florence-orange;
+  font-family: 'Merriweather';
+
+  @media (max-width: $screen-sm) {
+    text-align: left;
+  }
+}
+
+.attribution {
+  font-family: 'geomanist-medium';
+  font-size: 0.75em;
+
+  @media (max-width: $screen-sm) {
+    text-align: left;
+  }
+}
+
+#pill-bottle {
+  bottom: -25%;
+  left: 0;
+  position: absolute;
+  width: 17%;
+  z-index: 3;
+
+  @media (max-width: $screen-sm) {
+    bottom: 7%;
+    left: -20%;
+    max-height: 200px;
+    width: 50%;
+  }
+}
+
+#pill-bottle2 {
+  bottom: -15%;
+  left: 10%;
+  position: absolute;
+  width: 13%;
+  z-index: 1;
+
+  @media (max-width: $screen-sm) {
+    width: 30%;
+    bottom: 12%;
+    left: 3%;
+  }
+}
+
+#pill-bottle3 {
+  bottom: -15%;
+  left: -10%;
+  position: absolute;
+  width: 19%;
+  z-index: 0;
+
+  @media (max-width: $screen-sm) {
+    display: none;
+  }
+}
+
+/*----------- Key Factors -----------*/
+
+#key-factors {
+  color: #504949;
+  margin-top: 10%;
+
+  h3 {
+    color: $florence-orange;
+    font-size: 0.8em;
+    margin-bottom: -8px;
+    text-transform: uppercase;
+  }
+}
+.section-title {
+  color: $florence-blue;
+  font-size: 0.65em;
+  letter-spacing: 2px;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.section-intro {
+  color: $florence-blue;
+  font-family: 'Merriweather';
+  margin: auto;
+  text-align: center;
+  width: 47%;
+}
+
+#body {
+  display: block;
+  margin: 5% auto;
+  width: 20%;
+}
+
+.factor-box {
+  background-color: #ffffff;
+  border-radius: 9px;
+  font-size: 0.7em;
+  line-height: 1.4em;
+  padding: 25px;
+  position: relative;
+  width: 250px;
+
+  &-right {
+    border-radius: 37% 8px 8px 37%;
```

Comment:
The percentage values here might end up looking off if the text length or box size changes. I'd recommend changing it to `999px`. That'll get you an exact circle on that side, as long as the box isn't bigger than 500px tall. (same for the border radius properties below)